panoramix/assets/javascripts/dashboard.js

```diff
@@ -170,19 +173,51 @@ var Dashboard = function (dashboardData) {
         gridster.remove_widget(li);
       });
 
-      $(".slice_info").click(function () {
-        var widget = $(this).parents('.widget');
-        var slice_description = widget.find('.slice_description');
-        slice_description.slideToggle(500, function () {
-          widget.find('.refresh').click();
-        });
+      $("li.widget").click(function (e) {
```

Comment:
FYI this is typically better practice for event listeners, you place the listener on the parent element and the event "bubbles" up from the dom element you click on, up to the parent with the listener. then you don't have to add a million listeners.